cli/public_key/service.py

```diff
@@ -25,6 +26,25 @@ def __init__(self, service):
         """
         self.service = service
 
+    def get(self, address):
+        """
+        Get information about public key address by public key address.
+        """
+        try:
+            public_key_info = loop.run_until_complete(
+                self.service.public_key_storage.get_info(public_key_address=address),
+            )
+
+        except RpcGenericServerDefinedError as error:
+            return None, str(error.message)
+
+        except Exception as error:
+            return None, str(error)
+
+        return {
+            'public_key_info': public_key_info.data,
+        }, None
```

Comment:
```python
return public_key_info.data, None
```

The additional keys such as `public_key_info` are useful when you execute a command but the output does not say anything for you.

For example, you do `account transfer-tokens`. The result is a batch identifier. If you do not create key `batch_identifier`, the user won't understand it is a batch identifier because of it just a sequence.